packages/assets/scss/01-atoms/_fixed-feedback-button.scss

```
@@ -11,7 +11,7 @@
   margin-top: 0 !important;
   position: fixed;
     right: 32px; // the height of the button before rotation
-    bottom: 18rem; // tested with the longest translation option
+    top: 50%; // fixes the button in the verticle middle of the screen
   transform-origin: top right;
   transform: rotate(-90deg);
```

```suggestion
  transform: rotate(-90deg)  translateX(100%);
```
To position it at absolute center to ensure the button width is accounted for